### PR TITLE
1.10.1 master mergeup

### PR DESCRIPTION
**Pull request by branan**

### CODE REVIEW

File: configs/components/rubygem-nokogiri.rb [hunk 1]
```diff
@@ -7,9 +7,9 @@
     # :gem_home were then tar'ed up and packed into this container tarball.
     pkg.url "http://buildsources.delivery.puppetlabs.net/nokogiri-precompiled-huaweios-#{pkg.get_version}-for-ruby-#{settings[:ruby_version]}.tar.gz"
     if settings[:ruby_version] == "2.1.9"
-      pkg.md5sum "2ad8b4dd69f6e2b296dfaba293e9cb11"
-    elsif settings[:ruby_version] == "2.3.3"
-      pkg.md5sum "443e5b3bea092cd955e125c8e8bda0d6"
+      pkg.md5sum "4bf7a53bbfa5593041f7698f4e56a8d1"
+    elsif settings[:ruby_version] == "2.3.1"
+      pkg.md5sum "d11bcd650dc4942aa198586f4ff04247"
```
Comment (joshcooper) [inline configs/components/rubygem-nokogiri.rb]: should we stay at nokogiri 2.3.3?

Comment (branan) [inline configs/components/rubygem-nokogiri.rb]: Oh, we probably need to rebuild nokogiri as part of this mergeup - the CVE fork had more recent rebuilds, but we haven't rebuild to 2.3.3 for master. @ScottGarman what do you think needs doing here?

**Comment by branan** [comment]
Travis failure is because we haven't yet flagged the release tickets as public from the release

**Comment by ScottGarman** [comment]
:+1:

**Comment by MikaelSmith** [comment]
I guess we can ignore the failing commit check.

**Comment by Magisus** [comment]
Closing in favor of the newer cleaner https://github.com/puppetlabs/puppet-agent/pull/1050.

**Pull request closed without merging**